src/main/markdown/doc/latest/DevGuideIE9.md

```diff
@@ -1,107 +0,0 @@
-IE9
```

Comment:
Typically I'd prefer to leave old files in place, even if we remove links to them, or add a banner "This document only applies to IE9, which reach End of Life on XXXX" so that anyone who does find an inbound link or some content that helps can still see it. 

With that said, some of the links on the page are broken (but not all). I'm open to just removing it if no one thinks there can be any value here.

pre-submit edit: Ha, it still isn't truly totally dead. https://en.wikipedia.org/wiki/Internet_Explorer_9

> However, as Windows Server 2008 is eligible for the paid ESU (Extended Security Updates) program, security updates from Microsoft for IE9 will be released until January 9, 2024 (for [Azure](https://en.wikipedia.org/wiki/Microsoft_Azure) customers), following which, only IE11 (the final version of Internet Explorer) will continue to be supported in any capacity.

Reply:
ok, let's wait until January 10, 2024 ... :-)
I will mark the page as outdated

Reply:
added a hint, that the content is outdated.